@xen-orchestra/proxy/src/app/mixins/backups/index.js

```diff
@@ -65,7 +65,7 @@ export default class Backups {
 
           const xapi = yield this.getXapi(xapis[xapiId])
           return xapi.getRecordByUuid(type, uuid)
-        }),
+        }).bind(this),
```

Comment:
I'm going to connect that separately because it's unrelated to this PR :slightly_smiling_face: